Pipfile

```diff
@@ -5,7 +5,7 @@ name = "pypi"
 
 [packages]
 nbconvert = ">=6.4.5"
-notebook = ">=6.4"
+notebook = ">=6.4, <7.0.0"
```

Comment:
please add a comment explaining what has changed. why is this needed?

Reply:
added